### PR TITLE
Simplify conditional logics in main.js

### PR DESCRIPTION
**Pull request by KumarManoj-S**
Hacktober fest contribution: https://github.com/btzr-io/jelly-beats/issues/167

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -14,21 +14,17 @@ function createMainWindow() {
...
-  if (isDevelopment) {
-    window.webContents.openDevTools()
-  }
+  isDevelopment && window.webContents.openDevTools();
 
-  if (isDevelopment) {
-    window.loadURL(`http://localhost:${process.env.ELECTRON_WEBPACK_WDS_PORT}`)
-  } else {
-    window.loadURL(
+  isDevelopment
```
Comment (btzr-io) [inline src/main/index.js]: I think we can simplify this to:
```
 window.loadURL(isDevelopment ? localUrl : formatedUrl);
```

File: src/main/index.js [hunk 2]
```diff
@@ -14,21 +14,17 @@ function createMainWindow() {
     },
   })
 
-  if (isDevelopment) {
-    window.webContents.openDevTools()
-  }
+  isDevelopment && window.webContents.openDevTools();
```
Comment (btzr-io) [inline src/main/index.js]: This doesn't look really necessary, can you at least add a comment explaining what it does?

File: src/main/index.js (hunk 1 above)
Comment (btzr-io) [reply src/main/index.js]: and also add a simple comment

Comment (KumarManoj-S) [reply src/main/index.js]: Yes @btzr-io. Sounds correct I will make the changes.

**Comment by btzr-io** [comment]
is this the same? -> Refactor URL selection logics #170

**Pull request closed without merging**